usr/share/rear/prep/USB/Linux-i386/350_check_usb_disk.sh

```diff
@@ -9,8 +9,8 @@ REAL_USB_DEVICE=$(readlink -f $USB_DEVICE)
 StopIfError "USB device '$USB_DEVICE' is not a block device"
 
 # Check if REAR-000 USB device is not accidentally mounted on other than $BUILD_DIR location.
-if grep -v $BUILD_DIR /proc/mounts | grep -q "^$REAL_USB_DEVICE"; then
-    Error "USB device '$REAL_USB_DEVICE' is already mounted."
+if res=( $(grep -v $BUILD_DIR /proc/mounts | grep "^$REAL_USB_DEVICE" ) ) ; then
+    Error "USB device '$REAL_USB_DEVICE' is already mounted on '${res[2]}'"
```

Comment:
Strange, I actually tested this one before:
```bash
$ res=( $(mount | grep '^/dev/sda1' ) ) ; echo "${res[2]}"
/boot/efi
```

Does your mount output look different?